index.html

```diff
@@ -44,7 +44,8 @@ <h2>GAME OVER!</h2>
       </section>
     </main>
     <footer class="game-footer">
-      <p>Made for Hacktoberfest by these <strong><a href="https://github.com/itaditya/got-death-quiz#contributors" target="_blank">awesome people.</a></strong></p>
+      <p>Made for Hacktoberfest by these <strong><a href="https://github.com/itaditya/got-death-quiz#contributors" target="_blank">awesome people.</a></strong><a href="https://twitter.com/intent/tweet?text=Game%20of%20Thrones%20Death%20Quiz&url=https%3A%2F%2Fgot-death-quiz.netlify.com&hashtags=roadtohacktober,hacktoberfest" target="blank" rel="noopener noreferrer"><img
+          style="vertical-align: bottom;margin-left: 10px;" src="/assets/tweet.png" alt="Share Page on Twitter" /></a></p>
```

Comment:
Move the styling to CSS file style.css. Also make the border-radius: 4px

Reply:
Ah sure! updated the code now. 
Thank you!